src/app/modules/webhook/webhook.service.ts

```diff
@@ -118,9 +125,9 @@ const logWebhookSuccess = (
 }
 
 // Logging for webhook failure
-export const logWebhookFailure = (
+const logWebhookFailure = (
```

Comment:
Normally, you should give a description of the `@param` or the `@return` value. If you have nothing more to add, probably the comment line can be deleted as well (because it's not adding any value)?

Reply:
This is a template reply by Github btw, idk why sound so paggro LOL

Reply:
added jsdoc